python/lib/adaguc/CGIRunner.py

```diff
@@ -23,6 +24,94 @@
 sem = asyncio.Semaphore(int(ADAGUC_NUMPARALLELPROCESSES))
 
 
+ADAGUC_FORK_UNIX_SOCKET = f"{os.getenv('ADAGUC_PATH')}/adaguc.socket"
```

Comment:
I think it would be good to be able to set the socket path with an absolute value so it does not have to be in the adaguc-server-master directory